tools/toolchain/dbuild

```diff
@@ -206,6 +206,12 @@ EOF
     # to /etc/containers/containers.conf
 fi
 
+# Forward authentication, to avoid docker hub unauthenticated rate limits.
+auth_json="${HOME}/.config/containers/auth.json"
```

Comment:
we use the command `podman/docker login` and there is nothing store in auth.json so i don't think this will work

Reply:
> But podman login stores the credentials in auth.json, no?
> 
> I guess there's also ${XDG_RUNTIME_DIR}/containers/auth.json, so I should pick the first.

`${XDG_RUNTIME_DIR}/containers/auth.json` this will work